build.cmd

```diff
@@ -0,0 +1,16 @@
+call "C:\Apache Software Foundation\apache-maven-3.5.0\bin\mvn" clean package
```

Comment:
True to some extent. The other side of the coin is that readme files are never kept up to date with build script changes and not available for the different dev platforms.. Anything that helps and does not hinder is worth keeping.